configuration/config-txt/pi4-hdmi.md

```diff
@@ -0,0 +1,24 @@
+# Raspberry Pi 4 HDMI pipeline
+
+In order to support dual displays, and modes up to 4k60, the Raspberry Pi 4 has updated the HDMI composition pipeline hardware in a number of ways. One of the major changes is that it generates 2 output pixels for every clock cycle.
+
+Every HDMI mode has a list of timings that control all the parameters around sync pulse durations. These are typically defined via a pixel clock, and then a number of active pixels, a front porch, sync pulse, and back porch for each of the horizontal and vertical directions. 
+
+Running everything at 2 pixels per clock means that the Pi4 can not support a timing where any of the horizontal timings are not divisible by 2. The firmware and Linux kernel will filter out any mode that does not fulfill this criteria.
```

Comment:
Doesn't seem clear why 1366 uses a horizontal timing that isn't divisible by 2? I guess that doesn't really matter though.

Reply:
That what the standard has chosen, no idea why. Timing choice was nothing to do with us.

Reply:
I just meant that 1366 appears to be divisible by 2, so it's not clear why this violates the rules :wink:  But @6by9 's comment below goes into much more detail...

Reply:
It's any of the horizontal timing parameters, hence having described porches etc before.